tests/integration/fns.sh

```
@@ -43,7 +43,8 @@ ENCLAVE_BIN=./packages/ciphernode/target/debug/enclave
 # Function to clean up background processes
 cleanup() {
     echo "Cleaning up processes..."
-    kill $(jobs -p) 2>/dev/null
+    jobs -p
+    [ -n "$(jobs -p)" ] && kill -9 $(jobs -p) 2>/dev/null || true
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)